playbooks/ocp-install-preparation.yml

```diff
@@ -64,6 +64,7 @@
       copy:
         dest: /etc/sysconfig/docker-storage-setup
         content: |
+          STORAGE_DRIVER="{{ docker_storage_driver | default('overlay2') }}"
```

Comment:
@cloudxabide i be dumb. this is redundent since the value is in the `defaults/main.yml` you can just do `STORAGE_DRIVER="{{ docker_storage_driver }}"`

Reply:
just my brain confsuing the 12 ways to do defaults in ansible....